app/routes/verifier.py

```diff
@@ -484,19 +484,29 @@ async def delete_proof(
 )
 async def get_credentials_by_proof_id(
     proof_id: str,
+    referent: Optional[str] = None,
+    limit: Optional[int] = Query(default=100, ge=0, le=10000),
+    offset: Optional[int] = Query(default=0, ge=0),
```

Comment:
Soz, one last nitpick! We can deduplicate by using the `limit_query_parameter` and `offset_query_parameter` in `app/util/pagination.py`. 

(They don't have the ge/le validation yet - I'll add it there)

Reply:
cool will do. Also adding the e2e tests